gym/core.py

```diff
@@ -60,28 +64,32 @@ def np_random(self, value: RandomNumberGenerator):
+
+        When end of episode is reached, you are responsible for calling :meth:`reset` to reset this environment's state.
+        Accepts an action and returns a tuple `(observation, reward, done, info)`.
+
+        The return type is a tuple of four elements:
+
+        - observation: this will be an element of the environment's :attr:`observation_space`.
+          This may, for instance, be a numpy array containing the positions and velocities of certain objects.
+        - reward: the amount of reward returned as a result of taking the action.
+        - done: A boolean value for if the episode has ended, in which case further :meth:`step` calls will return undefined results.
+          A done signal may be emitted for different reasons: Maybe the task underlying the environment was solved successfully,
+          a certain timelimit was exceeded, or the physics simulation has entered an invalid state.
+        - info: A dictionary that may contain additional information regarding the reason for a ``done`` signal.
+          `info` contains auxiliary diagnostic information (helpful for debugging, learning, and logging).
+          This might, for instance, contain: metrics that describe the agent's performance state, variables that are
+          hidden from observations, information that distinguishes truncation and termination or individual reward terms
+          that are combined to produce the total reward
 
         Args:
             action (object): an action provided by the agent
 
-        This method returns a tuple ``(observation, reward, done, info)``
-
         Returns:
-            observation (object): agent's observation of the current environment. This will be an element of the environment's :attr:`observation_space`. This may, for instance, be a numpy array containing the positions and velocities of certain objects.
-            reward (float) : amount of reward returned after previous action
-            done (bool): whether the episode has ended, in which case further :meth:`step` calls will return undefined results. A done signal may be emitted for different reasons: Maybe the task underlying the environment was solved successfully, a certain timelimit was exceeded, or the physics simulation has entered an invalid state. ``info`` may contain additional information regarding the reason for a ``done`` signal.
-            info (dict): contains auxiliary diagnostic information (helpful for debugging, learning, and logging). This might, for instance, contain:
-
-                - metrics that describe the agent's performance or
-                - state variables that are hidden from observations or
-                - information that distinguishes truncation and termination or
-                - individual reward terms that are combined to produce the total reward
+            `(observation, reward, done, info)`: A tuple containing the environment's current `observation`,
```

Comment:
Fix formatting please

![image](https://user-images.githubusercontent.com/15806078/167290045-a4c15e2b-c479-4717-8c78-fa16a66ea975.png)


Also, it seems that a list in Returns actually works fine with the website:
![image](https://user-images.githubusercontent.com/15806078/167290486-6c50dcbe-c905-4920-9a6d-d8326be38136.png)
So I would change it back